### ISSUE TITLE
Does this replace M5_BH1750FVI.h ?

### ISSUE THREAD

**Issue by Sarah-C** [m5stack/M5-DLight#3]
I was trying to use M5_BH1750FVI for my M5Stick C Plus, and it always hung on getLux.

It's only available as an archive:
https://webcache.googleusercontent.com/search?q=cache:STMNKlzspCEJ:https://github.com/m5stack/M5_BH1750FVI+&cd=1&hl=en&ct=clnk&gl=uk

This new version M5-DLight...... works correctly, no tweaks needed! Thanks!

I can't see any logarithm changes!  

Did you work on the old one? What changed?

**Comment by TinyuZhao** [comment]
Yes, we migrated the library to this for consistent naming.
The previous initialization function is defined as follows
`void begin(TwoWire *wire = &Wire, uint8_t sda = 0, uint8_t scl = 26, uint32_t freq = 4000000);`
The initialization function is now defined as
`void begin(TwoWire *wire = &Wire, uint8_t sda = SDA, uint8_t scl = SCL, uint32_t freq = 4000000UL);`
After the change is made, different SDA and SCL values will be set depending on the device.

**Comment by Sarah-C** [comment]
I've had a look again after a good sleep - I saw the "readLux" originally opened a transaction and closed it again....   the new version doesn't use a transaction, and being curious I found taking those lines out of the old code stopped the code from freezing!

I guess the light meter doesn't wrap the constant lux reporting anymore in transactions...  I'd never found that out myself. Thanks for updating the library!  =D
